src/Core/Framework/Store/Subscriber/ExtensionChangedSubscriber.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Store\Subscriber;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Store\Services\StoreClient;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Contracts\Cache\CacheInterface;
+
+/**
+ * @internal
+ */
+#[Package('checkout')]
+readonly class ExtensionChangedSubscriber implements EventSubscriberInterface
+{
+    public function __construct(
+        private CacheInterface $cache
+    ) {
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            'plugin.written' => 'onExtensionChanged',
```

Comment:
we have this `\Shopware\Core\Framework\Plugin\PluginEvents::PLUGIN_WRITTEN_EVENT` but no equivalent for apps :see_no_evil:

Reply:
Shouldn't it be a DAL event, always triggering on table updates?

Reply:
I will create one. better to have it uniform I think.

Reply:
those are generated events by the DAL in the format `<entity_name>.written`. we introduced those constants in an event class for each entity very early. but didn't forced this with new entities :see_no_evil: